tests/remote_attestation/maa/sgxlkl_enclave/makefile

```diff
@@ -0,0 +1,29 @@
+.PHONY: all build clean alpine-container alpine-binaries lkl-image lkl-run
+
+SGXLKL_BIN_ROOT="$(SGXLKL_PREFIX)/bin"
+BUILD_DIR="/src/maa/sgxlkl_enclave"
+SGXLKLAPP_COMMON="/src/common"
+
+build-alpine-exe = sudo gcc -g -o $(BUILD_DIR)/alpine_sgxLklApp -fshort-wchar $(BUILD_DIR)/sgxLklApp.c $(BUILD_DIR)/clientAgent_init.c $(BUILD_DIR)/tlscli.c $(SGXLKLAPP_COMMON)/log_utils.c $(SGXLKLAPP_COMMON)/auth.cpp $(SGXLKLAPP_COMMON)/maa.cpp $(SGXLKLAPP_COMMON)/transport_utils.c $(SGXLKLAPP_COMMON)/curl_helper.cpp $(SGXLKLAPP_COMMON)/json_utils.c $(BUILD_DIR)/oe_private/verify_cert.c $(BUILD_DIR)/oe_private/cert.c -lmbedtls -lmbedx509 -lmbedcrypto -ldl -lcurl
+
+
+all: lkl-image
+
+clean:
+	rm -f clientAgent.o alpine_sgxLklApp sgxlkl-image.img*
+
+alpine-container:
+	docker build -t alpine3.10.build ./alpine
+
+alpine-binaries: alpine-container
+	docker run --rm  -v $(CURDIR)/../..:/src alpine3.10.build bash -c "$(build-alpine-exe)"
```

Comment:
Why are you doing this in two separate images? Is there any reason why you're not using multi-stage builds like in the other tests?

Reply:
Will fix that as Maik recommended.